encryption.rst

```diff
@@ -177,7 +314,7 @@ Running a container encrypted with a passphrase
 A passphrase can be supplied at runtime by either of the ways listed in
 the sections above.
 
-Running with a passphrase interactively
+Running with a passphrase interactively 
```

Comment:
Please remove the trailing blank.

Reply:
thanks for the comments. reflected in this commit https://github.com/apptainer/apptainer-userdocs/pull/179/commits/c7f08f6f219f7edc65f6a48fdae506166718c9e3